src/Features/Core/Portable/EditAndContinue/EditSession.cs

```diff
@@ -346,7 +346,7 @@ internal async Task<ImmutableArray<ActiveStatementExceptionRegions>> GetBaseActi
 
                     case CommittedSolution.DocumentState.OutOfSync:
                         var descriptor = EditAndContinueDiagnosticDescriptors.GetDescriptor(EditAndContinueErrorCode.DocumentIsOutOfSyncWithDebuggee);
-                        outOfSyncDiagnostics.Add(Diagnostic.Create(descriptor, Location.Create(document.FilePath, textSpan: default, lineSpan: default), new[] { document.FilePath }));
+                        outOfSyncDiagnostics.Add(Diagnostic.Create(descriptor, Location.Create(document.FilePath!, textSpan: default, lineSpan: default), new[] { document.FilePath! }));
```

Comment:
💡 If you allow null arguments, the second suppression can be removed:

```suggestion
                        outOfSyncDiagnostics.Add(Diagnostic.Create(descriptor, Location.Create(document.FilePath!, textSpan: default, lineSpan: default), new[] { document.FilePath }));
```

Reply:
Ah okay. This is where null arguments force my hand. 

---
In reply to: [356890250](https://github.com/dotnet/roslyn/pull/40268#discussion_r356890250) [](ancestors = 356890250)